DEFI/ConstantProductPool/certora/spec/ConstantProductPool.spec

```diff
@@ -202,15 +202,17 @@ rule monotonicityOfMint(uint256 x, uint256 y, address recipient) {
     env eT0;
     env eM;
     setup(eM);
+    address a;
+    require a == _token0 || a == _token1;
```

Comment:
rename 'a' to  'token' to make it more readable

Reply:
Done